### PR TITLE
Add new available fields + Node Properties

### PR DESCRIPTION
**Pull request by rachkor**
This PR adds a few quick fixes to the Burp addon: 

- Adds `references` and `vulnerability_classifications` as available fields to Burp plugin. Resolves: https://github.com/dradis/dradis-ce/issues/128

- Adds `hostname` as a Node property, eliminating the old `#[HostInfo]#` Notes

- Fixes formatting errors including `<p>`, `<a href="">`, and `<table>` tags

### CODE REVIEW

File: spec/burp_upload_spec.rb [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-      expect_any_instance_of(Dradis::Plugins::TemplateService)
+      expect_any_instance_of(TemplateService)
       .to receive(:default_templates_dir).and_return(templates_dir)
 
       # Init services
       plugin = Dradis::Plugins::Burp
 
-      @content_service = Dradis::Plugins::ContentService.new(plugin: plugin)
-      template_service = Dradis::Plugins::TemplateService.new(plugin: plugin)
+      @content_service = Dradis::Plugins::ContentService::Base.new(
```
Comment (dormi) [inline spec/burp_upload_spec.rb]: no need to call `Dradis::Plugins` all the time if the spec is in the same module?

**Comment by dormi** [comment]
we may include `<references>` also in `issue.sample` template

**Comment by dormi** [comment]
@rachkor note that there is a conflict with the base branch

File: spec/burp_upload_spec.rb (hunk 1 above)
Comment (rachkor) [reply spec/burp_upload_spec.rb]: Updated!

**Comment by rachkor** [comment]
@dormi I updated the `issue.sample` and fixed the conflict. Let me know if you see anything else?

File: lib/burp/issue.rb [hunk 2]
```diff
@@ -106,19 +107,22 @@ def cleanup_html(source)
       result.gsub!(/<h2>(.*?)<\/h2>/, '*\1*')
       result.gsub!(/<i>(.*?)<\/i>/, '\1')
       result.gsub!(/<p>(.*?)<\/p>/, '\1')
+      result.gsub!(/<p>/, '')
```
Comment (etdsoft) [inline lib/burp/issue.rb]: why is this needed? And shouldn't it be covered by line #109 ?

**Comment by etdsoft** [comment]
@rachkor minor comment. Also, for posterity I'd update the PR's description to mention the "vuln classifications" field.

**Comment by etdsoft** [comment]
Also missing:

- version bump
- CHANGELOG

File: lib/burp/issue.rb (hunk 2 above)
Comment (rachkor) [reply lib/burp/issue.rb]: @etdsoft there are instances like this where the `<p>` tags are on a separate line and aren't caught by Line #109: https://github.com/dradis/dradis-burp/blob/21eaec6b5310782329303af31353ff7b118ab599/templates/issue.sample#L11. Is there a better way to deal with these that I didn't think of?

**Comment by rachkor** [comment]
@etdsoft Updated the PR's description. I bumped the tiny version number here: https://github.com/dradis/dradis-burp/pull/9/commits/f08082c28c90b4f599a66df9c6b721b5e79a3a03. Do I need to bump the major version # too?

File: lib/burp/issue.rb (hunk 2 above)
Comment (etdsoft) [reply lib/burp/issue.rb]: what about using a multi-line shallow regexp?

`/<p>(.*?)<\/p>/m`

You can play with test data and regexp's here:

http://rubular.com/

**Comment by etdsoft** [comment]
> Do I need to bump the major version # too?

Probably that's a good idea.

Before merging, let's see if you can simplify the regex stuff

**Comment by aapomm** [comment]
@rachkor the changelog has conflicts to be resolved (due to the release bump)
